### ISSUE TITLE
File names

### ISSUE THREAD

**Issue by w64bit** [rikka0w0/ExplorerContextMenuTweaker#3]
Is it possible to change the file names, in order to be arranged in System32 folder one next to the other?
Like:
ECMT_Main
ECMT_ShellPayload

**Comment by rikka0w0** [comment]
Yes, it is possible. The main dll file can be renamed to anything, the payload dll has to be "ShellPayload.dll", if you want to change it, you can change it in the code and recompile everything:
https://github.com/rikka0w0/ExplorerContextMenuTweaker/blob/d289e3bbad1c156fb9730deda15ad54a98f17a11/MenuPatcher.cpp#L430
